### DISCUSSION TITLE
detailed skin

### DISCUSSION THREAD

**Discussion by DiggyDre** [lllyasviel/Fooocus#2862] [Q&A]
How do you get detailed skin in Fooocus? I understand that there is a refiner. The problem there is I am using a checkpoint model of a specific person. The refiner makes me lose the detail of who that person is. It isn't so bad when I use an SDXL refiner but that still doesn't really add realistic skin.

**Comment by mashb1t** [comment]
You can always use inpainting to add details. Can you please provide more information about what exactly you're struggling with, e.g. a screenshot?

**Comment by wisdombox44** [comment]
Try with some Realistic Models RealisticVisionXL, EpicRealismXL, NewRealityXL, LeoSam HelloWorlXL,
Add to your prompt realistic skin, diffused lightning, add to your negative promt, plastic skin, fake skin

**Reply by mashb1t** [reply]
Looks quite realistic to me, doesn't it? Please find the metadata attached to the image.

![image - 2024-05-04T170752 034](https://github.com/lllyasviel/Fooocus/assets/9307310/72d1f2f0-ff29-4ee4-b741-58b722754394)
![image - 2024-05-04T170641 187](https://github.com/lllyasviel/Fooocus/assets/9307310/4ac6f432-8614-45f8-aeeb-7f160e583515)

**Reply by ashailrizvi** [reply]
> Looks quite realistic to me, doesn't it? Please find the metadata attached to the image.
> 
Could you please share the log file. I'm not sure how to see the metadata attached to the image. Clicking on the image takes me to an inaccessible site.

**Reply by Ckrotky** [reply]
@ashailrizvi Use ☑️ Input Image > Metadata > Drag and drop the photos that **mashb1t** posted.
You'll see all the information you need to generate the same photo

you can apply all the settings for yourself by clicking [**Apply Metadata**] button

**Reply by mashb1t** [reply]
@ashailrizvi nevertheless, here you go

<details><summary>image 1</summary>
<p>

```
{
"prompt": "a woman with dyed hair and blue makeup looking into the distance, realistic skin, diffused lighting",
"negative_prompt": "unrealistic, saturated, high contrast, big nose, painting, drawing, sketch, cartoon, anime, manga, render, CG, 3d, watermark, signature, label, plastic skin, fake skin",
"prompt_expansion": "a woman with dyed hair and blue makeup looking into the distance, realistic skin, diffused lighting, highly detailed, sharp focus, fine detail, pretty, cozy, beautiful, intricate, elegant, warm light, cinematic, striking, new, color, epic, rich deep colors, aesthetic, joyful, vibrant, elaborate, complex, artistic, symmetry, great composition, clear",
"styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"performance": "Speed",
"resolution": "(896, 1152)",
"guidance_scale": 3,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "realisticStockPhoto_v20.safetensors",
"refiner_model": "None",
"refiner_switch": 0.5,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "304104460311784522",
"lora_combined_1": "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors : 0.25",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.4.1"
}
``` 

</p>
</details> 

<details><summary>image 2</summary>
<p>

```
{
"prompt": "realistic skin, diffused lightnin",
"negative_prompt": "unrealistic, saturated, high contrast, big nose, painting, drawing, sketch, cartoon, anime, manga, render, CG, 3d, watermark, signature, label, plastic skin, fake skin",
"prompt_expansion": "realistic skin, diffused lightnin, great composition, highly detailed, sharp focus, cinematic, complex, elegant, new, unique, attractive, creative, color bright, amazing, awesome, inspiring, wonderful, fine, full, pretty, surreal, intricate, artistic, beautiful, pure, cool, iconic, extremely expressive, friendly, cute, best, perfect, colorful",
"styles": "['Fooocus V2', 'Fooocus Photograph', 'Fooocus Negative']",
"performance": "Speed",
"resolution": "(896, 1152)",
"guidance_scale": 3,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "realisticStockPhoto_v20.safetensors",
"refiner_model": "None",
"refiner_switch": 0.5,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "1181254711170716649",
"lora_combined_1": "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors : 0.25",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.4.1"
}
```

</p>
</details>
